parser/parser_test.go

```diff
@@ -213,12 +225,16 @@ func TestFromFile(t *testing.T) {
 			assert.Equal(t, "MissingItem", operationState.OnErrors[1].Transition.NextState)
 			assert.Equal(t, "Missing order quantity", operationState.OnErrors[2].ErrorRef)
 			assert.Equal(t, "MissingQuantity", operationState.OnErrors[2].Transition.NextState)
-		}, "./testdata/workflows/checkinbox.cron-test.sw.yaml": func(t *testing.T, w *model.Workflow) {
+		},
+		"./testdata/workflows/checkinbox.cron-test.sw.yaml": func(t *testing.T, w *model.Workflow) {
+			assert.Equal(t, "Check Inbox Workflow", w.Name)
```

Comment:
We should organize the testcase later, with all field assert？

Reply:
not sure what do you mean.

Reply:
Sorry for did not make it clear. Should we change the testcode to:
```
"./testdata/workflows/checkinbox.cron-test.sw.yaml": func(t *testing.T, w *model.Workflow) {
			assert.Equal(t,  expectWorkflowObject, w)
}
```
, maybe this is more effective, it can verify all of workflow fields.